src/main/java/com/google/growpod/data/Plant.java

```diff
@@ -14,13 +14,22 @@
 
 package com.google.growpod.data;
 
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Entity.Builder;
+import com.google.cloud.datastore.Key;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
 /** Plant data class. */
+@Data
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
```

Comment:
default access level is public, no need to specify it

Reply:
Will address.